src/com/haskforce/features/intentions/AddToImports.scala

```diff
@@ -0,0 +1,92 @@
+package com.haskforce.features.intentions
```

Comment:
This is the largest file, and the one that does the most interesting thing:

It tries to add the import to the list of imports.

There are various cases not being considered yet:
1. A completely empty import list
2. Imports from modules which are already imported as qualified should not add an import but modify the not found import and prefix the qualified import
3. A few where I'm not sure if they even need treatment (hiding, open imports) because I don't think that an symbol should even show up as "not found" in the first place for these cases.